src/main/js/MergeModal.tsx

```diff
@@ -22,27 +22,36 @@
  * SOFTWARE.
  */
 import React from "react";
-import { Button, Modal, SubmitButton } from "@scm-manager/ui-components";
-import { WithTranslation, withTranslation } from "react-i18next";
+import {Button, Modal, SubmitButton} from "@scm-manager/ui-components";
+import {WithTranslation, withTranslation} from "react-i18next";
 import MergeForm from "./MergeForm";
-import { MergeCommit, PullRequest } from "./types/PullRequest";
-import { Link } from "@scm-manager/ui-types";
-import { getDefaultCommitDefaultMessage } from "./pullRequest";
+import {MergeCheck, MergeCommit, PullRequest} from "./types/PullRequest";
+import {Link} from "@scm-manager/ui-types";
+import {getMergeStrategyInfo} from "./pullRequest";
+import styled from "styled-components";
+import {MergeStrategyInfo} from "./types/MergeStrategyInfo";
```

Comment:
Your formatter is still broken. Whenever you change a lot of lines where you haven't changed things intentionally, this should trigger a warning.